src/icons/emojis/anxiousEmoji.ts

```diff
@@ -0,0 +1,5 @@
+import AnxiousEmojiPrimary from './svgs/anxious-emoji.svg';
+import { useIcon, IconProps } from '../../shared-components/icon';
+
+export default (props: IconProps) =>
```

Comment:
The default behavior of `create-index` is to generate an `index.ts` file comprising `default` exports renamed as their "safe" named export value. It automatically picks up that this should be `export { default as AnxiousEmoji }` from the filename. 

There's good React reasons to not use anonymous default exports--the component shows up as `Anonymous` in the Component tree in dev tools--but since the underlying components in `useIcon` will be named, this seemed a fine trade-off to make for enhanced compatibility with `create-index`. Moreover, It reduces auto-import name-spacing noise a bit. (Currently if you add `AnxiousEmoji` in a file, the auto-import will suggests both `../../src/icons` and `../../src/icons/emojis` as relative imports.)